index.html

```diff
@@ -1453,19 +1455,22 @@ <h2>
           </tr>
           <tr>
             <td>
-              <dfn>[[\serializedModifierData]]</dfn>
+              <dfn data-export="" data-dfn-for=
+              "PaymentRequest">[[\serializedModifierData]]</dfn>
             </td>
             <td>
               A list containing the serialized string form of each <a data-lt=
               "PaymentDetailsModifier.data">data</a> member for each
               corresponding item in the sequence
-              <a>[[\details]]</a>.<a data-lt="PaymentDetailsBase">modifier</a>,
-              or null if no such member was present.
+              [=PaymentRequest/[[details]]=].<a data-lt=
```

Comment:
We can probably fix this one: {{PaymentDetailsBase/modifiers}} ... might need to look this one up in the other spec to see if this is correct.

Reply:
oops, I mean in this spec.

Reply:
as discussed - gonna do this in a separate PR cos this PR is just for exporting terms. also, there are 150+ `data-lt` things. XD